Mathlib/Topology/UniformSpace/UniformConvergenceTopology.lean

```diff
@@ -1118,3 +1118,27 @@ instance {α β : Type*} [UniformSpace β] [CompleteSpace β] : CompleteSpace (
   (UniformOnFun.uniformEquivUniformFun β {univ} (mem_singleton _)).completeSpace_iff.1 inferInstance
 
 end UniformFun
+
+section UniformComposition
+
+universe u v w x
+
+variable {α : Type u} {β : Type v} {γ : Type w} {ι : Type x} [UniformSpace β] [UniformSpace γ]
+  {F : ι → α → β} {f : α → β} {s s' : Set α} {x : α} {p : Filter ι} {p' : Filter α} {g : ι → α}
```

Comment:
It seems like at least `F`, `f`, `s`, `s'`, and `x` are unused. Please check the others too